docs/prms/mincostflow.rst

```diff
@@ -11,6 +11,19 @@ MinCostFlow takes two optional parameters:
 * flow: (int) the amount of flow going through the graph
 * capacity: the (float) max capacity for edges
 
+Dataset Usage
+-------------
+
+MinCostFlow uses the input's ``sources``, ``targets``, and edge weights. MinCostFlow also
```

Comment:
MCF will only accept a fully directed graph, and will output an undirected graph.